### ISSUE TITLE
Disk Space parsing Bitcoin data

### ISSUE THREAD

**Issue by makeholer** [norestlabs/mousexplore-vcoins#331]
I'm parsing all bitcoin data but im on block 230k and the mongo already have almost 120gb. From what I calculate it will need almost 3terabytes to storage everything, is there any changes i can do to store only critical data? (like for addresses it would be ok to store only the transactions txid instead of all the transaction info, i think this is what consume more space).

Please help, I'm not familiar with mongodb + node and after reading all the files I cant find on how to do those changes.

**Comment by gedanziger** [comment]
are you checking only the mongodb size? Ours isn't taking up 3TB space.

**Comment by makeholer** [comment]
Yes, the current database is bitcoin-db as you can see on this pic https://imgur.com/a/nXddYlz

But I'm not running the node on prune mode, maybe that's the difference between us?

**Comment by oleg-baturov** [comment]
in mousebelt btc node, `chain data` is 227GB.

mongodb size is not big.
![image](https://user-images.githubusercontent.com/32242962/49563084-f6112080-f969-11e8-8780-6132bdabee76.png)

**Comment by makeholer** [comment]
Your explorer doesn't store everything, as you can see for example our address 1BrasiLb2KMbdtuhb1chAVnS2FvcNGfV9J has only 28 transactions on http://mousexplore.mousebelt.com/btc/address/1BrasiLb2KMbdtuhb1chAVnS2FvcNGfV9J but on blockchain it have more than 10k. But maybe there's nothing that can be done to store all blockchain data with low disk space, I'll try to improve my droplet.
